### PR TITLE
Swift 3.0 - Xcode 8 Beta 4

### PR DESCRIPTION
**Pull request by acegreen**
Conforms To Xcode 8 Beta 4

### CODE REVIEW

**Comment by acegreen** [comment]
@liuxuan30 whoever reviews this should pay a closer look because beta 4 complained about complex calculations to I had to break it down, and could be a cause of miscalculation 

For example:
```
let tempPOW = pow(2.0, -10.0 * position)
let tempSIN = sin((position * duration - s) * (2.0 * M_PI) / p) 
return CGFloat( tempPOW * tempSIN * 0.5 + 1.0 )
```

File: Charts/Classes/Renderers/BarChartRenderer.swift [hunk 1]
```diff
@@ -557,7 +557,10 @@ public class BarChartRenderer: ChartDataRendererBase
 							context.saveGState()
 							
 							let pixelToValueMatrix = trans.pixelToValueMatrix
-							let xToYRel = abs(sqrt(pixelToValueMatrix.b * pixelToValueMatrix.b + pixelToValueMatrix.d * pixelToValueMatrix.d) / sqrt(pixelToValueMatrix.a * pixelToValueMatrix.a + pixelToValueMatrix.c * pixelToValueMatrix.c))
+                            let sqrtAC = sqrt(pixelToValueMatrix.a * pixelToValueMatrix.a + pixelToValueMatrix.c * pixelToValueMatrix.c)
```
Comment (liuxuan30) [inline Charts/Classes/Renderers/BarChartRenderer.swift]: indent issue

**Comment by liuxuan30** [comment]
in BarChartRenderer.swift, the indent is not correct. Others seem good to go. Please fix the indent then I will merge.
If you are willing to practice squash skill with SourceTree, this is the best time. Or you can make another commit, it's fine.

**Comment by liuxuan30** [comment]
no worry, I will take care of it.

**Pull request merged**